bootstrap/eks/api/v1alpha3/eksconfigtemplate_types.go

```diff
@@ -28,6 +28,13 @@ type EKSConfigTemplateSpec struct {
 	Template EKSConfigTemplateResource `json:"template"`
 }
 
+// EKSConfigTemplateStatus defines the observed state of EKSConfigTemplate
+type EKSConfigTemplateStatus struct {
```

Comment:
can we get rid of this type entirely? i think it's unused now.

Reply:
Removed: https://github.com/newrelic-forks/cluster-api-provider-aws/pull/13/commits/8bd1a585852f7d86e74fca19cccb44a1847c6e8d